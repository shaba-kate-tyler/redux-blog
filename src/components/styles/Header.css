.Header {
    display: flex;
    justify-content: space-around;
    background-color: rgb(217,83,79);
    font-family: 'Roboto', sans-serif;
    padding: 1.5vh;
    a {
        text-decoration: none;
        font-size: 4vh;
        font-weight: 600;
        color: white;
    }
    a.animate {
        text-decoration: none;
        color: white;
        position: relative;
    }
    a.animate:after {
        content: '';
        position: absolute;
        top: 103%;
        left: 0;
        width: 0%;
        border-bottom: 2px solid rgb(170, 63, 61);
        transition: 0.2s;
    }
    a.animate:hover:after {
        width: 100%;
    }
}

