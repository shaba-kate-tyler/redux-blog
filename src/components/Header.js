import React from 'react';
import { Link } from 'react-router-dom';
import styles from './styles/Header.css';

export default function Header() {
  return (
    <div className={styles.Header}>
      <Link to="/" className={styles.animate}>Home</Link>
      <Link to="/users" className={styles.animate}>Users</Link>
    </div>
  );
}
