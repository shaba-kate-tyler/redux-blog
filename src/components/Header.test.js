import React from 'react';
import renderer from 'react-test-renderer';
import Header from './Header';
import { MemoryRouter } from 'react-router-dom';

describe('Header', () => {
  it('matches a snapshot', () => {
    const tree = renderer.create(
      <MemoryRouter>
        <Header />
      </MemoryRouter>
    ).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
