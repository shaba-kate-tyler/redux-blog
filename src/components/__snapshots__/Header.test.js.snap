// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header matches a snapshot 1`] = `
<div
  className="Header"
>
  <a
    className="animate"
    href="/"
    onClick={[Function]}
  >
    Home
  </a>
  <a
    className="animate"
    href="/users"
    onClick={[Function]}
  >
    Users
  </a>
</div>
`;
